src/App.tsx

```diff
@@ -4,9 +4,10 @@ import Game from './interfaces/Game'
 import LibrariesPage from './pages/LibrariesPage';
 import LibraryPage from './pages/LibraryPage';
 import GamePage from './pages/GamePage';
+import GameInstancePage from "./pages/GameInstancePage";
```

Comment:
Does it make sense maybe to just have one game page and then just conditionally render the additional custom user information if it exists? I imagine otherwise we'll have a fair amount of overlap between the two